server/issue.go

```diff
@@ -25,8 +25,10 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 	api := ji.GetPlugin().API
 
 	create := &struct {
-		PostId string           `json:"post_id"`
-		Fields jira.IssueFields `json:"fields"`
+		PostId         string           `json:"post_id"`
+		AttachedToPost bool             `json:"attached_to_post"`
```

Comment:
nit/naming: suggest `s/AttachedToPost/FromPost/`

Reply:
Also, shouldn't `PostId != ""` (and subsequently `post != nil` be enough? Do we really need a separate flag?

Reply:
Good point. I usually like to make execution paths explicit, but `create.PostId != ""` should be clear enough.